resource/js/term-counts.js

```diff
@@ -0,0 +1,45 @@
+const termCountsApp = Vue.createApp({
+  data () {
+    return {
+      languages: []
+    }
+  },
+  mounted () {
+    fetch('https://api.finto.fi/rest/v1/yso/labelStatistics?lang=fi') /* What should this be? */
```

Comment:
This should be a relative URL. In Skosmos 2.x, the variable `rest_base_url` was defined in a script [config.js](https://github.com/NatLibFi/Skosmos/blob/31f430b65be209d55209d53729c6beb7c92df984/resource/js/config.js) and then all AJAX style calls used that as the base URL.

For now I think we could just hardwire this to `rest/v1/...` i.e. strip the `https://api.finto.fi/` URL prefix and it should work.